README.md

```diff
@@ -262,29 +262,48 @@ in the changelog when publishing a new release.
 For simplicity of deployment, versioning of packages are fixed and updated
 together.
 
-#### Publishing a new version of a package
+#### Publishing a new version of a package (3.0 and later)
```

Comment:
So long as we maintain all the packages in this repo, I think directions should include how to lint, test, and deploy all packages in this repo.